examples/classic_controllers/classic_controllers.py

```diff
@@ -2,16 +2,39 @@
 from gym_electric_motor.physical_systems import SynchronousMotorSystem, DcMotorSystem, DcSeriesMotor, \
     DcExternallyExcitedMotor
 from gym_electric_motor.reference_generators import MultipleReferenceGenerator, SwitchedReferenceGenerator
-from gym_electric_motor.envs import ContDcExternallyExcitedMotorEnvironment, DiscDcExternallyExcitedMotorEnvironment, ContPermanentMagnetSynchronousMotorEnvironment
+from gym_electric_motor.visualization import MotorDashboard
+from gym_electric_motor import envs
```

Comment:
As it seems this import does not make use of the "local" GEM files within the same directory but rather necessitates that GEM is installed in the used Python environment. 
This complicates testing of new code and should therefore be avoided.

Reply:
(If this is the case it applies to all files of this pull request.)